ui/src/main/java/com/adyen/checkout/ui/internal/InternalColors.kt

```diff
@@ -18,6 +18,7 @@ import com.adyen.checkout.ui.theme.CheckoutColors
 @Immutable
 data class InternalColors(
     val background: Color,
+    val backgroundInverse: Color,
```

Comment:
What do you think of naming this `shadow` to make it more explicit? Or are there other use cases for this color?